circuits/cpp/barretenberg/cpp/src/barretenberg/stdlib/primitives/bit_array/bit_array.test.cpp

```diff
@@ -50,10 +52,8 @@ TYPED_TEST(BitArrayTest, test_uint32_input_output_consistency)
 
     EXPECT_EQ(result.size(), 2UL);
 
-    auto a_result =
-        static_cast<uint32_t>(composer.get_variable(result[0].get_witness_index()).from_montgomery_form().data[0]);
-    auto b_result =
-        static_cast<uint32_t>(composer.get_variable(result[1].get_witness_index()).from_montgomery_form().data[0]);
+    auto a_result = static_cast<uint32_t>(result[0].get_value());
```

Comment:
With the simulator we can't get variable values, but anyway, the stdlib tests should not access builder functions.